lib/configs/configs.dart

```diff
@@ -0,0 +1,19 @@
+const clientId = '<client_id>';
+const redirectUrl = 'wso2.asgardeo.flutterapp://login-callback';
+const organizationName = '<organization-name>';
+const spName = '<sp-name>';
+const externalAPIEndpoint = '<external-api-url>';
+const accessURL = 'wso2.asgardeo.flutterapp://signup';
+
+//optional
+const defaultPhotoURL = '<default-photo-url>';
+
+
+const asgardeoLogo = "assets/images/asgardeo-logo-transparent.png";
+const flutterLogo = "assets/images/flutter5786.jpg";
+const repoURL = "https://github.com/Dinithi-Hasanika/Asgardeo-Flutter-APP";
+const gitHubLogo = "assets/images/GitHub-Mark.png";
+
+//styles
+const primaryColor = 0xffFF7300;
```

Comment:
Also we can create a theme to manage primary colours. Not mandatory.
https://docs.flutter.dev/cookbook/design/themes

Reply:
theme is included, primary color is able to change as per user's choice